pages/inventory.js

```diff
@@ -150,6 +150,12 @@ export default function Inventory() {
     ref.once("value")
     .then(function(resp) {
       let res = resp.val();
+      // Insert new col "Case Count"
+      for (const key in res) {
+        res[key].caseCount = ((parseInt(res[key].packSize) !== 1) && (parseInt(res[key].count) > 1))
```

Comment:
I agree we should only display case count when count is positive! It's currently possible for a packSize to be negative so we should probably enforce it being positive either when adding/modifying an item, otherwise case count could be negative which is a little weird

Reply:
Gotcha! If count is neg, I'll display case count as zero.